src/core/killall.c

```diff
@@ -31,13 +31,14 @@
 #include "parse-util.h"
 #include "process-util.h"
 #include "set.h"
+#include "stat-util.h"
 #include "string-util.h"
 #include "terminal-util.h"
 #include "util.h"
 
 #define TIMEOUT_USEC (10 * USEC_PER_SEC)
 
-static bool ignore_proc(pid_t pid) {
+static bool ignore_proc(pid_t pid, bool killing_spree) {
```

Comment:
`killing_spree` doesn't mean much, you have to read the code to see what it means. Maybe call it something like `warn_rootfs`, and call this function as 
```
ignore_proc(pid, sig == SIGKILL && !in_initrd())
```
?